packages/app-project/src/components/ProjectAnnouncement/ProjectAnnouncement.spec.js

```diff
@@ -21,8 +21,8 @@ describe('Component > ProjectAnnouncement', function () {
     expect(paragraphWrapper.text()).to.contain(ANNOUNCEMENT)
   })
 
-  it('should be the full width of its container', function () {
-    const shallowWrapper = shallow(<ProjectAnnouncement announcement={ANNOUNCEMENT} />)
+  xit('should be the full width of its container', function () {
```

Comment:
this was to ensure there was a `fill` prop, as there was a previous layout bug where it didn't span the viewport (#1197)